src/fu-util.c

```diff
@@ -4367,6 +4381,47 @@ fu_util_get_bios_setting(FuUtilPrivate *priv, gchar **values, GError **error)
 	return TRUE;
 }
 
+static gboolean
+fu_util_fix_security(FuUtilPrivate *priv, gchar **values, GError **error)
+{
+	if (g_strv_length(values) == 0) {
```

Comment:
don't you need a check for HSI like you have in the fu`-tool.c`?

Reply:
Ohh yes, indeed we do. Thanks, fixed.